### ISSUE TITLE
Loss is nan, stopping training

### ISSUE THREAD

**Issue by Scout7666** [jiaowoguanren0615/MobileNetV4#7]
I'm a binary task, in the use of ’mobilenetv4_hybrid_medium', 'mobilenetv4_hybrid_large‘ is 45epoch and 23epoch respectively Loss is nan, stopping training, is there any way to solve this problem?

**Comment by jiaowoguanren0615** [comment]
In my personal experience, if you use pretrained weights, you need to set a lower learning_rate, like: 1e-4 or 2e-5. Moreover, you can set a suitable batch-size for training, try to take up 80% of the GPU memory. Finally, I add a MSAG module in model, you can re-clone my code and train your dateset. I hope that can solve your problem.

**Comment by Scout7666** [comment]
你好你好，我通过这些代码训练了我的数据集得到了相应权重，想在此询问您，如何利用我得到的权重进行数据集的预测工作？感谢您的回复!

**Comment by jiaowoguanren0615** [comment]
> 你好你好，我通过这些代码训练了我的数据集得到了相应权重，想在此询问您，如何利用我得到的权重进行数据集的预测工作？感谢您的回复!

您好，train_gpu.py里面，predict参数默认为True的呀，所以等你训练完成之后，在你没有修改predict的情况下，他会打印出classification_report以及保存的confusion matrix图片，还有roc图片，都是png格式。
![image](https://github.com/user-attachments/assets/6326e824-3561-44f1-9c8c-a569e15255f1)

**Comment by jiaowoguanren0615** [comment]
> 你好你好，我通过这些代码训练了我的数据集得到了相应权重，想在此询问您，如何利用我得到的权重进行数据集的预测工作？感谢您的回复!

你可以看一下目录里有没有两个png文件
